packages/fast-components-react-base/src/image/image.spec.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from "react";
+import * as ShallowRenderer from "react-test-renderer/shallow";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, shallow } from "enzyme";
+import examples from "./examples.data";
+import { generateSnapshots } from "@microsoft/fast-jest-snapshots-react";
+import {
+    IImageClassNameContract,
+    IImageHandledProps,
+    IImageManagedClasses,
+    IImageUnhandledProps,
+    ImageProps,
+} from "./image";
+
+/*
+ * Configure Enzyme
+ */
+configure({adapter: new Adapter()});
+
+describe("image", (): void => {
+    generateSnapshots(examples);
+});
+
+describe("image unit-tests", (): void => {
+    const Component: React.ComponentClass<IImageHandledProps & IImageManagedClasses> = examples.component;
+    const managedClasses: IImageClassNameContract = {
+        image: "image-class",
+        image_round: "image-round",

```

Comment:
did you want this to be image-round-class and picture-class? seems to be what you are doing elsewhere.